### PR TITLE
define namespaced elements

### PR DESCRIPTION
**Pull request by WilcoFiers**
First version of this. It needs a to be used in a bunch more places, but before I do that work I'd like to hear what people think of the approach. It's a little different from what was discussed on last week's call.

Need for Final Call: 1 week

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: pages/glossary/namespaced-element.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Namespaced Element
+key: namespaced-element
+unambiguous: true
+objective: true
+---
+
+An [element][] with a specific [namespaceURI][] value from [HTML namespaces][]. For example an "SVG element" is an element with the "SVG namespace", which is `http://www.w3.org/2000/svg`.
+
+Namespaced elements are not limited to elements described in a specification. They also include custom elements. Elements such as `a`, `title`, and `svg` have a different namespace depending on where they are used. For example a `title` in an HTML page usually has the HTML namespace. When used in an `svg` element, a `title` elements has the SVG namespace instead.
```
Comment (carlosapaduarte) [inline pages/glossary/namespaced-element.md:10]:
```suggestion
Namespaced elements are not limited to elements described in a specification. They also include custom elements. 
Elements such as `a`, `title`, and `svg` have a different namespace depending on where they are used. For example a `title` in an HTML page usually has the HTML namespace. When used in an `svg` element, a `title` element has the SVG namespace instead.
```

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
I'm okay with this. But I'm failing to understand how does this address what we discussed in the call?

**Comment by Jym77** [comment]
> I'm okay with this. But I'm failing to understand how does this address what we discussed in the call?

That bit: https://www.w3.org/2021/05/27-act-rules-minutes.html#t03 / #459, 
Not that bit: https://www.w3.org/2021/05/27-act-rules-minutes.html#t04 / #1593

**Comment by carlosapaduarte** [comment]
> > I'm okay with this. But I'm failing to understand how does this address what we discussed in the call?
> 
> That bit: https://www.w3.org/2021/05/27-act-rules-minutes.html#t03 / #459,
> Not that bit: https://www.w3.org/2021/05/27-act-rules-minutes.html#t04 / #1593

Thanks for pointing it out! I was looking in the wrong place 👍

File: pages/glossary/namespaced-element.md (hunk 1 above)
Comment (Jym77) [inline pages/glossary/namespaced-element.md:10]: I actually think that a `<svg>` element inside an HTML document is in the SVG namespace…

[`svg` element in HTML specs](https://html.spec.whatwg.org/multipage/embedded-content-other.html#svg-0) links to [SVG `foreignObject`](https://svgwg.org/svg2-draft/embedded.html#ForeignObjectElement) which explains it's here for inclusion in other XML docs and has a note saying:
> The HTML parser treats elements inside the ‘`foreignObject`’ equivalent to elements inside an HTML document fragment. Any `svg` or `math` element, and their descendents, will be parsed as being in the SVG or MathML namespace, respectively;

**Review by Jym77 — CHANGES_REQUESTED** [review]

File: pages/glossary/namespaced-element.md (hunk 1 above)
Comment (WilcoFiers) [reply pages/glossary/namespaced-element.md:10]: I prefer to keep this as is.

Comment (WilcoFiers) [reply pages/glossary/namespaced-element.md:10]: Good catch! You are right, `svg` has the SVG namespace in HTML.

Comment (WilcoFiers) [inline pages/glossary/namespaced-element.md:10]:
```suggestion
Namespaced elements are not limited to elements described in a specification. They also include custom elements. Elements such as `a` and `title` have a different namespace depending on where they are used. For example a `title` in an HTML page usually has the HTML namespace. When used in an `svg` element, a `title` elements has the SVG namespace instead.
```

File: pages/glossary/namespaced-element.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Namespaced Element
+key: namespaced-element
+unambiguous: true
+objective: true
+---
+
+An [element][] with a specific [namespaceURI][] value from [HTML namespaces][]. For example an "SVG element" is an element with the "SVG namespace", which is `http://www.w3.org/2000/svg`.
+
+Namespaced elements are not limited to elements described in a specification. They also include custom elements. Elements such as `a` and `title` have a different namespace depending on where they are used. For example a `title` in an HTML page usually has the HTML namespace. When used in an `svg` element, a `title` elements has the SVG namespace instead.
```
Comment (carlosapaduarte) [inline pages/glossary/namespaced-element.md:10]:
```suggestion
Namespaced elements are not limited to elements described in a specification. They also include custom elements. Elements such as `a` and `title` have a different namespace depending on where they are used. For example a `title` in an HTML page usually has the HTML namespace. When used in an `svg` element, a `title` element has the SVG namespace instead.
```

**Review by carlosapaduarte — DISMISSED** [review]
typo

**Review by Jym77 — APPROVED** [review]

**Review by colabottles — APPROVED** [review]

**Review by ajanec01 — APPROVED** [review]

**Review by tbostic32 — APPROVED** [review]

**Comment by WilcoFiers** [comment]
After this one's merged, I'm going to update other uses of "HTML" and "SVG". I'll do that in a separate PR.

**Review by colabottles — APPROVED** [review]

**Pull request merged**